### PR TITLE
Scripts/Mission: Added Cait Sith mission.

### PR DESCRIPTION
**Pull request by Vivitarut**

### CODE REVIEW

File: scripts/zones/Batallia_Downs_[S]/npcs/Cavernous_Maw.lua [hunk 1]
```diff
@@ -46,7 +55,11 @@ end;
 function onEventFinish(player,csid,option)
     -- printf("CSID: %u",csid);
     -- printf("RESULT: %u",option);
-    if (option == 1) then
+	if (csid == 0x02BD) then
+		player:completeMission(WOTG, BACK_TO_THE_BEGINNING);
+		player:addMission(WOTG, CAIT_SITH);
+		player:setPos(-44,0,428,191,84);
+    elseif (option == 1) then
```
Comment (xdemolish) [inline scripts/zones/Batallia_Downs_[S]/npcs/Cavernous_Maw.lua]: should check the csid here too

File: scripts/zones/Sauromugue_Champaign_[S]/npcs/Cavernous_Maw.lua [hunk 2]
```diff
@@ -45,8 +54,12 @@ end;
...
 function onEventFinish(player,csid,option)
     -- printf("CSID:",csid);
-    -- printf("RESULT:",option);
-    if (option == 1) then
+    -- printf("RESULT:"..option);
+    if (csid == 0x02BD) then
+		player:completeMission(WOTG, BACK_TO_THE_BEGINNING);
+		player:addMission(WOTG, CAIT_SITH);
+		player:setPos(369,8,-229,0,98);
+    elseif (option == 1) then
```
Comment (xdemolish) [inline scripts/zones/Sauromugue_Champaign_[S]/npcs/Cavernous_Maw.lua]: same here

File: scripts/zones/Rolanberry_Fields_[S]/npcs/Cavernous_Maw.lua [hunk 3]
```diff
@@ -46,7 +56,11 @@ end;
 function onEventFinish(player,csid,option)
     -- printf("CSID:",csid);
     -- printf("RESULT:",option);
-    if (option == 1) then
+	if (csid == 0x02BD) then
+		player:completeMission(WOTG, BACK_TO_THE_BEGINNING);
+		player:addMission(WOTG, CAIT_SITH);
+		player:setPos(-204,7,365,13,91);
+    elseif (option == 1) then
```
Comment (xdemolish) [inline scripts/zones/Rolanberry_Fields_[S]/npcs/Cavernous_Maw.lua]: and here

**Comment by teschnei** [comment]
still waiting on the requested changes before merging this

**Comment by xdemolish** [comment]
reopen when requested changes have been made

**Pull request closed without merging**